tests/NLog.UnitTests/Targets/DatabaseTargetTests.cs

```diff
@@ -1005,11 +1108,9 @@ public MockDbParameter(MockDbCommand mockDbCommand, int paramId)
                 this.paramId = paramId;
             }
 
-            public DbType DbType
-            {
-                get { return this.parameterType; }
-                set { this.parameterType = value; }
-            }
+            public OracleDbType OracleDbType { get; set; }
```

Comment:
Add comments; property for Oracle as we don't have the Oracle types here. 

(or something like that)